tests/basic/pool_function_test.go

```diff
@@ -833,3 +834,87 @@ var _ = Describe("{PoolExpandAddDiskInMaintenanceMode}", func() {
 		})
 	})
 })
+
+var _ = Describe("{PoolVolUpdateResizeDisk}", func() {
+
+	var testrailID = 34542876
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542876
+
+	BeforeEach(func() {
+		StartTorpedoTest("PoolVolUpdateResizeDisk", "expand volume to the pool and pool expansion using resize-disk", nil, testrailID)
+		contexts = scheduleApps()
+	})
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		storageNode, err = GetNodeWithGivenPoolID(poolIDToResize)
+		log.FailOnError(err, "Failed to get node with given pool ID")
+
+	})
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+	stepLog := "Expand volume to the pool and pool expansion using resize-disk"
+	It(stepLog, func() {
+		var newRep int64
+		var currRep int64
+		poolToResize = getStoragePool(poolIDToResize)
```

Comment:
You have poolIDToResize in JustBeforeEach. It is not required in the start of this test("It") then.